dist/node-jsonapi-datastore.js

```diff
@@ -155,6 +156,23 @@
     _createClass(JsonApiDataStore, [{
       key: "destroy",
       value: function destroy(model) {
+        var self = this;
+        model._dependents.forEach(function(dependent, depIdx) {
+          self.graph[dependent.type][dependent.id]._dependents.forEach(function(val, idx) {
+            if (val.id === model.id) {
+              self.graph[dependent.type][dependent.id]._dependents.splice(idx, 1);
+            }
+          });
+          if (self.graph[dependent.type][dependent.id][dependent.relation].constructor === Array) {
```

Comment:
This part should be moved to a `_removeDependence(type, id, relation)` on the Model.